MoleculeVAE-reImplementation/train.py

```diff
@@ -307,13 +322,18 @@ def train(epochs,train_set_final, valid_set_final,logger):
                         torch.save(model.state_dict(), 'Weights/' + str(dt_string.replace("/", '_')) + str(epochs) + '_param.pth')
 
                     print(f'{epochs} / {batch_idx}\t{loss:.4f}')
-        return (train_loss.cpu().detach().numpy() / len(train_set_final)), (valid_loss.cpu().detach().numpy() / len(valid_set_final))
-
-def test(test_set,epochs,logger):
-    dataset=torch.utils.data.DataLoader(test_set, batch_size=params['batch_size'])
+        return (train_loss.cpu().detach().numpy() / len(train_dataloader)), (valid_loss.cpu().detach().numpy() / len(valid_dataloader))
+
+def test(
+        test_set: data.Dataset,
```

Comment:
I've typed the inputs here to try and avoid confusions about what should be passed in. This helps a good IDE flag is something unexpected is going into the function.